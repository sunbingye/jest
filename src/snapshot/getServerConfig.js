export const getServerConfig = () => {
    return { host: 'https://test.com', port: '88' };
};
