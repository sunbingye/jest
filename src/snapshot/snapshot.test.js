import { getServerConfig } from './getServerConfig.js';

// // test('快照测试 getServerConfig', () => {
// //     expect(getServerConfig()).toEqual({
// //         server: 'https://test.com',
// //         port: '80',
// //     });
// // });

// test('快照测试 getServerConfig', () => {
//     expect(getConfig()).toMatchSnapshot();
// });

test('快照测试 getServerConfig', () => {
    expect(getServerConfig()).toMatchSnapshot();
});
