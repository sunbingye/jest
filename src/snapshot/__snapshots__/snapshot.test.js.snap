// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`快照测试 getServerConfig 1`] = `
{
  "host": "https://test.com",
  "port": "88",
}
`;
